lib/AppInfo/Application.php

```diff
@@ -78,6 +78,7 @@ public function register(IRegistrationContext $context): void {
 			};
 			$config = $c->get(IConfig::class);
 			$allowRootShare = $config->getAppValue('groupfolders', 'allow_root_share', 'true') === 'true';
+			$enableEncryption = $config->getAppValue('groupfolders', 'enable_encryption', 'false') === 'true';
```

Comment:
I'm wondering if we should also bring this to the web UI as it will likely be a more common option than allow_root_share ?

Reply:
Yes, I am in favor. I would look for it there.